docs/guide/maven-plugins/goals/build-user-feature-pack/user-fp-build-xml/index.adoc

```diff
@@ -22,24 +22,15 @@ For example:
 
 [source,xml]
 ----
-<feature-pack xmlns="urn:jboss:galleon:feature-pack:2.0" location="org.foo:my-galleon-pack:1.0.0.Final">
-    <transitive>
-        <dependency location="wildfly-core@maven(org.jboss.universe:community-universe):current#9.0.1.Final">
-            <origin>org.wildfly.core:wildfly-core-galleon-pack</origin>
-        </dependency>
-        <dependency location="wildfly-servlet@maven(org.jboss.universe:community-universe):current#17.0.0.Final">
-            <origin>org.wildfly:wildfly-servlet-galleon-pack</origin>
-        </dependency>
-    </transitive>
+<build xmlns="urn:wildfly:feature-pack-build:2.0" producer="org.foo:my-galleon-pack:1.0.0.Final">
```

Comment:
I don't know why but with schema 2.0 the build is failing in OpenShift with error:
```
[ERROR] Failed to execute goal org.wildfly.galleon-plugins:wildfly-galleon-maven-plugin:4.0.4.Final:build-user-feature-pack (postgresql-datasources-galleon-pack-build) on project postgresql-datasources-galleon-pack: Failed to parse /tmp/src/postgresql-datasources-galleon-pack/wildfly-user-feature-pack-build.xml: ParseError at [row,col]:[2,134]
```

When I use `<build xmlns="urn:wildfly:feature-pack-build:3.0" ...` it seems working. Is usage of galleon implying need for 3.0+ ?

Reply:
Good catch, yes, the producer attribute is defined in 3.0. 3.0 needs to be used.

Reply:
OK, then the same needs to be updated here https://github.com/wildfly/galleon-plugins/blob/master/docs/guide/maven-plugins/wildfly-feature-pack-build-xml/index.adoc

Reply:
Right, fixed.